### PR TITLE
Feature static files

### PR DESCRIPTION
**Pull request by avoevodin**
#5

### CODE REVIEW

File: mysite/uwsgi.ini [hunk 1]
```diff
@@ -5,6 +5,9 @@ wsgi-file=%v/mysite/wsgi.py
 http-socket=0.0.0.0:8000
 workers=%k
 static-map=/static=%v/static
-enable-threads=True
+enable-threads=False
```
Comment (iliadmitriev) [inline mysite/uwsgi.ini]: Why did you disable threads?

**Review by iliadmitriev — CHANGES_REQUESTED** [review]

File: mysite/uwsgi.ini (hunk 1 above)
Comment (avoevodin) [reply mysite/uwsgi.ini]: I was just testing )

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,9 @@
 asgiref==3.3.1
+click==7.1.2
```
Comment (iliadmitriev) [inline requirements.txt]: to be deleted

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,9 @@
...
 coverage==5.5
 Django==3.1.7
+h11==0.12.0
```
Comment (iliadmitriev) [inline requirements.txt]: to be deleted

**Review by iliadmitriev — APPROVED** [review]

**Pull request merged**